src/components/ProductCard/ProductCard.stories.tsx

```diff
@@ -0,0 +1,82 @@
+import type { Meta, StoryObj } from '@storybook/react';
+
+import ProductCard from './ProductCard';
+import Tag from '../Tag/Tag';
+import Image from '../Image/Image';
+
+const meta = {
+  title: 'Components/Product card',
+  component: ProductCard,
+  argTypes: {
+    dropShadow: {
+      if: { arg: 'gradientBrand', truthy: false }
+    },
+    gradientBrand: {
+      if: { arg: 'dropShadow', truthy: false }
+    }
+  },
+  tags: ['autodocs']
+} satisfies Meta<typeof ProductCard>;
+
+export default meta;
+type Story = StoryObj<typeof ProductCard>;
+
+export const Playground: Story = {
+  args: {
+    texts: {
+      title: 'Product card title',
+      description:
+        'Lorem ipsum dolor sit amet, consectetur adipiscing elit. Duis nec vestibulum augue, viverra aliquet nunc. Cras eget felis sodales, vestibulum neque ac, rhoncus ipsum.'
+    },
+    linkTo: 'docs-demo-link',
+    headingLevel: 'h2'
+  }
+};
+
+export const SemanticCardTitle: Story = {
+  name: 'Semantic card title',
+  args: {
+    ...Playground.args
+  }
+};
```

Comment:
wondering if we need this one since we're presenting it as the default. can we add a TODO to check with catie? we may also want to specify non-semantic as the default if we're having to specify props as undefined.